### DISCUSSION TITLE
Yellow down arrow in process icon meaning?

### DISCUSSION THREAD

**Discussion by mega-optimus** [sandboxie-plus/Sandboxie#3023] [Documents and notes]
What does it mean? 
Thank you!
![image](https://github.com/sandboxie-plus/Sandboxie/assets/69918075/e6d81d23-2c99-4b8e-8340-052f140a0d0f)

**Comment by DavidXanatos** [comment]
This means that the process uses a restricted token, its often encountered when using electron apps, or chromium or firefox based browsers
there is also a orange arrow indicating the use of a appcontainer, it is only available when using a green sandbox

**Reply by isaak654** [reply]
This is the source code in question, if anyone wants to check it out:

https://github.com/sandboxie-plus/Sandboxie/blob/0c538d3b9d4949cec3e5a77f6035875d2b36f7a0/SandboxiePlus/SandMan/Models/SbieModel.cpp#L399-L408

[SandboxiePlus/SandMan/Resources/Actions/Restricted.png](https://github.com/sandboxie-plus/Sandboxie/blob/7477b7693522106e4ab7adbd386efa9b1c09ba5f/SandboxiePlus/SandMan/Resources/Actions/Restricted.png)

Below is the setting to disable the overlay icons (`Global Settings` -> `Interface Config` -> `User Interface`):

https://github.com/sandboxie-plus/Sandboxie/blob/0c538d3b9d4949cec3e5a77f6035875d2b36f7a0/SandboxiePlus/SandMan/Forms/SettingsWindow.ui#L947-L951
